stix_shifter_modules/cybereason/configuration/config.json

```diff
@@ -0,0 +1,44 @@
+{
+    "connection": {
+        "type": {
+            "displayName": "Cybereason"
+        },
+        "host": {
+            "type": "text",
+            "regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])$"
+        },
+        "port": {
+            "type": "number",
+            "default": 443,
+            "min": 1,
+            "max": 65535
+        },
+        "help": {
+            "type": "link",
+            "default": "data-sources.html"
+        },
+        "cert": {
+            "type": "password",
+            "optional": true
+        },
+        "sni": {
+            "type": "text",
+            "optional": true
+        },
+        "selfSignedCert": {
```

Comment:
do you need any of these parameter **cert/sni/selfSignedCert** in order to authenticate the API? If the answer is No, then these parameters should be removed from the config.json

Reply:
Removed the SignedCert , sni in Config.json, lang_en.json, api_client.py files